src/components/icons/_icons.scss

```diff
@@ -5,7 +5,7 @@
   fill: currentColor;
 
   .accent {
-    fill: var(--wxc-icon--active-color);
+    fill: var(--wxc-unmuted-color);
```

Comment:
please use --wxc-icon--accent-color, it should have the same value as --wxc-unmuted-color